src/sdk/src/fragment-components/a-slider.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import {
+	Slider,
+	TextInput,
+	NumberInput,
+	Checkbox
+} from 'carbon-components-react';
+import { css, cx } from 'emotion';
+import { AComponent, ComponentInfo } from './a-component';
+import image from './../assets/component-icons/slider.svg';
+import {
+	angularClassNamesFromComponentObj,
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj
+} from '../tools';
+
+const preventCheckEventStyle = css`
+	pointer-events: none;
+`;
+
+export const ASliderSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<Checkbox
+			labelText='Disabled'
+			id='disable-label'
+			checked={selectedComponent.disabled}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					disabled: checked
+				});
+			}} />
+
+		<Checkbox
+			labelText='Hide text input'
+			id='hide-text-input'
+			checked={selectedComponent.textInputIsHidden}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					textInputIsHidden: checked
```

Comment:
Just knitpicking, why not just name the variable `hideTextInput` to match the same prop name as React? It's shorter and means the same thing.